enarxbot-pr-assign

```diff
@@ -119,7 +119,11 @@ def get_responsible(pr):
     # The last event which assigns responsibility is the one that we choose.
 
     events = sorted(merge_events_and_reviews(pr), key=lambda x: x["when"])
-    author = pr['author']['login']
+    author = pr['author']
+    if len(author) > 0:
+        author = author['login']
+    else:
+        author = None
```

Comment:
Why not this?

`author = pr['author'].get('login')`

Reply:
Duh. Thanks for the suggestion.

Reply:
Fixed.